dotnetv3/StepFunctions/Actions/HelloStepFunctions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[StepFunctions.dotnetv3.HelloStepFunctions]
+namespace StepFunctionsActions;
+
+public class HelloStepFunctions
+{
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for AWS Step Functions.
+        using var host = Host.CreateDefaultBuilder(args)
+            .ConfigureServices((_, services) =>
+                services.AddAWSService<IAmazonStepFunctions>()
+            )
+            .Build();
+
+        var stepFunctionsClient = host.Services.GetRequiredService<IAmazonStepFunctions>();
+
+        Console.Clear();
+        Console.WriteLine("Welcome to AWS Step Functions");
+        Console.WriteLine("Let's list up to 10 of your state machines:");
+        var stateMachineListRequest = new ListStateMachinesRequest { MaxResults = 10 };
+
+        // Get information for up to 10 Step Functions state machines.
+        var response = await stepFunctionsClient.ListStateMachinesAsync(stateMachineListRequest);
+
+        if (response.StateMachines.Count > 0)
+        {
+            response.StateMachines.ForEach(stateMachine =>
+            {
+                Console.WriteLine($"\tActivity Name: {stateMachine.Name}\tCreated on: {stateMachine.CreationDate}");
```

Comment:
Is this the Activity Name or the State Machine Name? The spec says "Print the name and stateMachineArn of each state machine returned."

Reply:
Yes, it is the state machine name and I've changed the output to match the changed spec.